PowerAux/lib/src/BPSCANInterface.cpp

```diff
@@ -1,8 +1,8 @@
 #include "BPSCANInterface.h"
-#include "log.h"
+#include "Logging.h"
 
 BPSCANInterface::BPSCANInterface(PinName rd, PinName td, PinName standby_pin)
-    : CANInterface(rd, td, standby_pin) {
+    : CANInterface("BPSCANInterface", rd, td, standby_pin) {
```

Comment:
Could `#define BPSCANInterface_ID 2` and use ints to identify? I think might be a cleaner solution.